platform/go/go-std/instrumented/src/go.mod

```diff
@@ -0,0 +1,19 @@
+module go-std/src/instrumented
```

Comment:
A bit annoying, but I had to rename this because of [golang/go#45184](https://togithub.com/golang/go/issues/45184). Long story short, the vscode plugin for Go cannot handle multiple modules with the same name in the workspace.

The side effect is having to change import paths in all Go files to match this rename.